File: support_tools/swigwin-3.0.12/Examples/test-suite/octave/return_const_value_runme.m
return_const_value

p = return_const_value.Foo_ptr.getPtr();
if (p.getVal() != 17)
  error("Runtime test1 faild. p.getVal()=", p.getVal())
endif

p = return_const_value.Foo_ptr.getConstPtr();
if (p.getVal() != 17)
  error("Runtime test2 faild. p.getVal()=", p.getVal())
endif

File: support_tools/swigwin-3.0.12/Examples/test-suite/perl5/return_const_value_runme.pl
#!/usr/bin/perl
use strict;
use warnings;
use Test::More tests => 4;
BEGIN { use_ok('return_const_value') }
require_ok('return_const_value');

# adapted from ../python/return_const_value_runme.py

is(return_const_value::Foo_ptr::getPtr()->getVal(), 17);

is(return_const_value::Foo_ptr::getConstPtr()->getVal(), 17);

